SecurityManager/Clients.Web/UI/SecurityManagerAutoCompleteReferenceValueWebService.asmx.cs

```diff
@@ -94,6 +95,7 @@ public static void BindServiceToControl (BocAutoCompleteReferenceValue control)
 
       var businessObjectClassWithIdentity = GetBusinessObjectClassWithIdentity(businessObjectClass);
       var referenceProperty = GetReferenceProperty(businessObjectProperty, businessObjectClassWithIdentity);
+      // TODO how to add pagination here correctly
```

Comment:
our SecurityManager Service is going to ignore this feature, at least for now. 

please make a follow-up ticket to properly integrate with Linq-skip/take clauses.